### PR TITLE
WIP:  tests susemanager gui and others with different and multiple minions

### PR DESCRIPTION
**Pull request by MalloZup**
todo:
- [x] test current pr
- [x] change hostname of containers so we can see which os is that
- [x] test if sles11sp4 and non systemd system works 
- [ ] install pkg, run commad, and other op, run script with suse-mgr

### CODE REVIEW

**Comment by MalloZup** [comment]
@mbologna, @meaksh 

```for i in {1..5}; do docker run -d minion; done``` ( referring at the image)

but i had some troubles when i accepted the 5 keys on suma-srv. I couldn't see the minion ( after accepting the keys) 🤣

**Comment by mbologna** [comment]
@MalloZup did you implement the workaround for generating `machine-id` inside Docker container that we spoke about?

**Comment by MalloZup** [comment]
@mbologna  yes i implemented it 
 🏩 
https://github.com/MalloZup/mulinsale#for-testing

**Comment by MalloZup** [comment]
todo:
- [ ] test current pr
- [ ] change hostname of containers so we can see which os is that
- [ ] install pkg, run commad, and other op, run script with suse-mgr

File: features/step_definitions/command_steps.rb [hunk 1]
```diff
@@ -317,3 +317,21 @@
...
+And(/I create dockerized minions$/) do
+master, _code = $minion.run("cat /etc/salt/minion.d/susemanager.conf")
+## build everything
+$minion.run("docker build https://gitlab.suse.de/galaxy/suse-manager-containers.git#master:minion-fabric/rhel6/ -t rhel6", true, 2000)
+$minion.run("docker build https://gitlab.suse.de/galaxy/suse-manager-containers.git#master:minion-fabric/rhel7/ -t rhel7", true, 2000)
+$minion.run("docker build https://gitlab.suse.de/galaxy/suse-manager-containers.git#master:minion-fabric/sles11sp4 -t sles11sp4", true, 2000)
+$minion.run("docker build https://gitlab.suse.de/galaxy/suse-manager-containers.git#master:minion-fabric/sles12 -t sles12", true, 2000)
+$minion.run("docker build https://gitlab.suse.de/galaxy/suse-manager-containers.git#master:minion-fabric/sles12sp1 -t sles12sp1", true, 2000)
+# launch the key to master
+$minion.run("docker run -d --entrypoint '/bin/sh' rhel6 -c \"echo #{$MASTER} > /etc/salt/minion; dbus-uuidgen > /etc/machine-id; salt-minion -l trace\"")
```
Comment (MalloZup) [inline features/step_definitions/command_steps.rb]: i don't know if rhel6 has dbus-uuidgen command, how we track systems without machine-id file?

**Pull request merged**